app/src/test/java/org/pesmypetcare/mypetcare/controllers/community/TestTrLikePost.java

```diff
@@ -0,0 +1,66 @@
+package org.pesmypetcare.mypetcare.controllers.community;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.pesmypetcare.mypetcare.features.community.forums.Forum;
+import org.pesmypetcare.mypetcare.features.community.groups.Group;
+import org.pesmypetcare.mypetcare.features.community.posts.Post;
+import org.pesmypetcare.mypetcare.features.community.posts.PostAlreadyLikedException;
+import org.pesmypetcare.mypetcare.features.community.posts.PostNotFoundException;
+import org.pesmypetcare.mypetcare.features.users.User;
+import org.pesmypetcare.mypetcare.services.StubCommunityService;
+import org.pesmypetcare.usermanager.datacontainers.DateTime;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author Xavier Campos
+ */
+public class TestTrLikePost {
+    private TrLikePost trLikePost;
+    private User user;
+    private Post post;
+
+    @Before
+    public void setUp() {
+        trLikePost = new TrLikePost(new StubCommunityService());
+        user = new User("Manolo Lama", "bicholover@gmail.com", "1234");
+        Group group = new Group("Husky", "John Doe",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "John Doe" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5443034)